### ISSUE TITLE
Access level for files: Setting is Private but still says Open to Group

### ISSUE THREAD

**Issue by KevinChagnon** [gctools-outilsgc/gcconnex#702]
"When I import a file and choose Private Access, then I return to the file, it indicates that the access is Open to the Group even if it is not.

When I am ready to put the access Open to the Group, I have to re-chose Access to the group ... even if we already see it as Access to the group to provide Group access. When I have a lot of files to put live, this is difficult to manage, because I do not know if any file has been put live or if it is still private, etc. Moreover, the date does not change, when a file is replaced or if access is changed; this is another problem because users do not know if a file was updated. In this case, I write in the date myself in the description"

**Comment by ToferC** [comment]
@KevinChagnon - can you please confirm that you have verified the user issue and outline the help activities you undertook prior to adding them to GitHub? Thanks!

**Comment by KevinChagnon** [comment]
I have tested this myself.  I uploaded a file in a test group and set the Access to "only me".  I was able to see the file while logged in with the profile with which I uploaded the file, but not while logged in to another profile.  When I went to edit the Access level, it was displayed that the access level was set to "All public servants" but in reality it was still set to "only me".  When I used the drop down to select "All public servants" and then clicked save, the file was then accessible using the other test profile.  In settings it did show the Access level to be "All public servants"

In conclusion, when "only me" is selected and saved, when you return to the edit screen it will show as "all public servants" even though this is not the setting it currently holds.

The user expresses frustration not being able to know the actual Access level of her files when entering edit mode.

Hope this helps

**Comment by Phanoix** [comment]
This should have been fixed with the last update, can you verify this issue is resolved?

**Comment by KevinChagnon** [comment]
The reported issue "Setting is Private but still says Open to Group" is resolved.  The setting is now properly displayed and acts accordingly. However now, when you click "Save" you receive a message "**This field is required**" under the "Choose File" button when there is no need to upload a new file;

**Comment by Troy-Lawson** [comment]
@KevinChagnon is this still an issue? can we close this

**Comment by Steph4104** [comment]
Related gctools-outilsgc/gcconnex#1095
